fdbserver/TLogServer.actor.cpp

```diff
@@ -3240,10 +3251,19 @@ ACTOR Future<Void> restorePersistentState(TLogData* self,
 						logData->knownCommittedVersion =
 						    std::max(logData->knownCommittedVersion, qe.knownCommittedVersion);
 						if (qe.version > logData->version.get()) {
-							commitMessages(self, logData, qe.version, qe.arena(), qe.messages);
+							std::set<Tag> tags;
+							commitMessages(self, logData, qe.version, qe.arena(), qe.messages, tags);
 							logData->version.set(qe.version);
 							logData->queueCommittedVersion.set(qe.version);
 
+							if (SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST) {
```

Comment:
I think this block is not needed. The `tLogCore()` is only invoked at the end of this function, which accepts peeks. Thus, there should be no `waitingTags` at this point.